server/drivers/curses_drv.c

```diff
@@ -705,25 +699,27 @@ curses_wborder (Driver *drvthis)
 
 
 static chtype
-get_color_by_name (char *colorname, chtype default_color) {
-	if (strcasecmp(colorname, "red") == 0)
-		return COLOR_RED;
-	else if (strcasecmp(colorname, "black") == 0)
-		return COLOR_BLACK;
-	else if (strcasecmp(colorname, "green") == 0)
-		return COLOR_GREEN;
-	else if (strcasecmp(colorname, "yellow") == 0)
-		return COLOR_YELLOW;
-	else if (strcasecmp(colorname, "blue") == 0)
-		return COLOR_BLUE;
-	else if (strcasecmp(colorname, "magenta") == 0)
-		return COLOR_MAGENTA;
-	else if (strcasecmp(colorname, "cyan") == 0)
-		return COLOR_CYAN;
-	else if (strcasecmp(colorname, "white") == 0)
-		return COLOR_WHITE;
-
-	return default_color;
+map_color_to_curses (CursesColor color) {
+	switch(color) {
+		case CURSES_COLOR_RED:
```

Comment:
Why does the code not directly use the generated enum?

Reply:
The used values (`COLOR_RED`, etc) come from the curses library.